README.textile

```diff
@@ -200,10 +200,9 @@ We have just covered a very small portion of what Elasticsearch is all about. Fo
 
 h3. Building from Source
 
-Elasticsearch uses "Maven":http://maven.apache.org for its build system.
+Elasticsearch uses "Gradle":http://gradle.org for its build system. You'll need to have a modern version of Gradle installed - 2.7 should do.
 
-In order to create a distribution, simply run the @mvn clean package
--DskipTests@ command in the cloned directory.
+In order to create a distribution, simply run the @gradle clean build@ command in the cloned directory.
```

Comment:
clean isn't really necessary (I have run like maybe once ever couple days, but only because I am severly mucking with tasks setup). But I guess it doesn't hurt to have it here..I just don't think it should normally be run (have to stop thinking about maven brokeness that causes us to run clean all the time).